src/main/g8/application/src/main/scala/$package__packaged$/api/Server.scala

```diff
@@ -44,7 +44,7 @@ object Server extends IOApp {
         )
       ).orNotFound
       server <- {
-        BlazeServerBuilder[IO]
+        BlazeServerBuilder[IO](connectionEc)
```

Comment:
This is surprising to me -- why should the server builder use the same context as database interaction?

Reply:
@jisantuc lol I used whatever first to make compiler happy.